x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/index/RestrictedIndicesNames.java

```diff
@@ -9,19 +9,27 @@
 import org.apache.lucene.util.automaton.Automaton;
 import org.elasticsearch.common.util.set.Sets;
 import org.elasticsearch.xpack.core.security.support.Automatons;
-import org.elasticsearch.xpack.core.upgrade.IndexUpgradeCheckVersion;
 
 import java.util.Collections;
 import java.util.Set;
 
 public final class RestrictedIndicesNames {
-    public static final String AUDIT_INDEX_NAME_PREFIX = ".security_audit_log";
-    public static final String INTERNAL_SECURITY_INDEX = ".security-" + IndexUpgradeCheckVersion.UPRADE_VERSION;
+    public static final String INTERNAL_SECURITY_INDEX_6 = ".security-6";
+    public static final String INTERNAL_SECURITY_INDEX_7 = ".security-7";
     public static final String SECURITY_INDEX_NAME = ".security";
 
-    public static final Set<String> NAMES_SET = Collections.unmodifiableSet(Sets.newHashSet(SECURITY_INDEX_NAME, INTERNAL_SECURITY_INDEX));
+    /**
+     * A set of names for the security indices
+     */
+    public static final Set<String> SECURITY_INDICES = Collections.unmodifiableSet(
+        Sets.newHashSet(SECURITY_INDEX_NAME, INTERNAL_SECURITY_INDEX_6, INTERNAL_SECURITY_INDEX_7));
+
+    /**
+     * All restricted indices (currently only security indices, but there could be more in the future)
```

Comment:
Remove the extra variable until we need it ?

Reply:
I went back and forth on this. In the end I'm not fussed either way.

I added the distinction for the purposes of tests. Some tests want to do things with all (or randomly one) of the security indices, other tests need to check each of the restricted indices.

But I'm happy to switch it. It was somewhat arbitrary.

Reply:
> I added the distinction for the purposes of tests. Some tests want to do things with all (or randomly one) of the security indices, other tests need to check each of the restricted indices.

I see the value now, I also think that it's worth leaving it as is or add a comment in the tests to be updated once more restricted indices are added.